jcl/src/openj9.sharedclasses/share/classes/com/ibm/oti/shared/SharedClassURLClasspathHelper.java

```diff
@@ -271,4 +271,9 @@
 	 * <p>
 	 */
 	public void confirmAllEntries();
+	
+	/**
+	 * <p>Notify the open/close state to all the jar/zip files on the URL classpath of this URLClassLoader.</p>
+	 */
+	public void notifyURLClasspathStateChange(boolean isOpen);
```

Comment:
As mentioned in https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/317
I'm not convinced this change should be added, I don't think the feature works properly. If the jar file is used by multiple classloaders, removing the CPM_ZIP_OPEN flag will cause timestamp checks to start occurring every time for the remaining classloaders using the file.

Reply:
All related code here has been removed.